recipes/flock/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: flock
+  version: "1.0"
+source:
+  git_url: https://github.com/cristhomas/immport-test.git
```

Comment:
Please specify the git_rev or use a tarball.
Btw. why is this different from the sourceforge page?